backend/app/rest/api/rest_private.go

```diff
@@ -39,11 +41,19 @@ type private struct {
 	imageService     *image.Service
 	notifyService    *notify.Service
 	authenticator    *auth.Service
+	telegramService  TelegramService
 	remarkURL        string
 	anonVote         bool
 	templates        templates.FileReader
 }
 
+// TelegramService is a subset of Telegram service used for setting up user telegram notifications
+type TelegramService interface {
```

Comment:
is there a reason for the public interface here? this thing constructed privately anyway